packages/react-components/src/components/CaptionsBanner.tsx

```diff
@@ -142,12 +182,37 @@ export const CaptionsBanner = (props: CaptionsBannerProps): JSX.Element => {
       scrollToBottom();
     }
   }, [captions, isAtBottomOfScroll]);
+  /* @conditional-compile-remove(rtt) */
+  const [textFieldValue, setTextFieldValue] = useState<string>('');
+  /* @conditional-compile-remove(rtt) */
+  useEffect(() => {
+    // if the latest real time text sent by myself is final, clear the text field
+    // find the last real time text caption
+    const lastRealTimeText = captions
```

Comment:
do we think the selector could do this calculation for us instead?

Reply:
Yep! Updated